sections/infrastructure.include

```diff
@@ -34,13 +34,13 @@
   objects and their descendant DOM trees, and to serialized byte streams using the
   <a href="#syntax">HTML syntax</a> or <a href="#xhtml">XHTML syntax</a>, depending on context.
 
-  In the context of the DOM structures, the terms <dfn>HTML document</dfn> and <a>XML document</a>
-  are used as defined in the DOM specification, and refer specifically to two different modes that
-  {{Document}} objects can find themselves in. [[!DOM]] (Such uses are always hyperlinked to their
-  definition.)
+  In the context of the DOM structures, the terms <dfn data-export="">HTML document</dfn>
```

Comment:
Per [the Bikeshed docs](https://tabatkins.github.io/bikeshed/#dfn-export), this need only be the Boolean attribute `export`, rather than `data-export=""`